README.md

```diff
@@ -42,10 +42,15 @@ To get a local copy up and running follow these simple example steps.
 - It can be used to make simple maths calculations.
 - It can be used to read a random math-related quote.
 
-## Author
+## Authors
```

Comment:
- [ ] Your all test passing. However, I'm facing a tiny error when I opened the app in localhost. 
It shows me:

> Module not found: Error: Cannot find file: 'button.js' does not match the corresponding name on disk: './src/components/Button.js'.

Kindly fix this error.

Reply:
Thank you very much sir, will do